src/battle_anim.c

```diff
@@ -319,7 +300,23 @@ void LaunchBattleAnimation(u32 animType, u32 animId)
 
     sMonAnimTaskIdArray[0] = TASK_NONE;
     sMonAnimTaskIdArray[1] = TASK_NONE;
-    sBattleAnimScriptPtr = animsTable[animId];
+
+    switch (animType)
+    {
+    case ANIM_TYPE_GENERAL:
+    default:
+        sBattleAnimScriptPtr = gBattleAnims_General[animId];
+        break;
+    case ANIM_TYPE_MOVE:
+        sBattleAnimScriptPtr = gMovesInfo[animId].battleAnimScript;
```

Comment:
Should default to some placeholder script e g. Tackle if battleAnimScript is NULL. Also might be good to add a debug print warning in that situation

Reply:
✔️